nautobot/dcim/api/serializers.py

```diff
@@ -545,6 +556,8 @@ def get_field_names(self, declared_fields, info):
         fields = list(super().get_field_names(declared_fields, info))
         self.extend_field_names(fields, "parent_bay")
         self.extend_field_names(fields, "config_context", opt_in_only=True)
+        if "cluster" not in fields:
+            fields.append("cluster")
```

Comment:
Could this use `self.extend_field_names()` instead, that is if it's even necessary at all?

Reply:
This is accomplished above